rest_framework/utils/field_mapping.py

```diff
@@ -106,6 +106,15 @@ def get_field_kwargs(field_name, model_field):
                               isinstance(model_field, models.TextField)):
         kwargs['allow_blank'] = True
 
+    if isinstance(model_field, models.FilePathField):
+        kwargs.update({
+            'path': model_field.path,
+            'match': model_field.match,
+            'recursive': model_field.recursive,
+            'allow_files': model_field.allow_files,
+            'allow_folders': model_field.allow_folders
+        })
```

Comment:
Do we want to exclude any of these if they have the default? (Partly for the purposes of a simple serializer repr)